Directory.Build.props

```diff
@@ -5,6 +5,7 @@
   <Import Project="build/Dependencies.props" />
 
   <PropertyGroup>
+    <DefaultTestTargetFramework Condition="'$(DefaultTestTargetFramework)' == ''">netcoreapp2.1</DefaultTestTargetFramework>
```

Comment:
This doesn't belong here. It is a "test" only property. It should be in the `test` directory (and it already is [https://github.com/dotnet/machinelearning/blob/master/test/Directory.Build.props#L5](https://github.com/dotnet/machinelearning/blob/master/test/Directory.Build.props#L5)).